.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI Checks
+on:
+  push:
+    branches: ["**"]
```

Comment:
what does this exactly? ["**"]

Reply:
and why is it necessary?

Reply:
This mean run this check on a push to any branch. Useful when we develop changes in our fork.

Reply:
Isn't this the default?